src/operator/tensor/la_op.cc

```diff
@@ -550,5 +550,79 @@ NNVM_REGISTER_OP(_backward_linalg_gelqf)
 .set_attr<nnvm::TIsBackward>("TIsBackward", true)
 .set_attr<FCompute>("FCompute<cpu>", LaOpBackward<cpu, 2, 2, 4, 1, gelqf_backward>);
 
+NNVM_REGISTER_OP(_linalg_syevd)
+.add_alias("linalg_syevd")
```

Comment:
linalg_* alias was added for previous ops for backward compatibility. New ops don't need this alias.
_linalg_syevd enables access from mx.sym.linalg.syevd

Reply:
Great to see we can now use the linalg namespace.

Does this work for all operators, or just for the new ones?
Is it already reflected in the documentation?